### ISSUE TITLE
use aws-keychain for credentials

### ISSUE THREAD

**Issue by mbrannigan** [garnaat/kappa#84]
I don't usually write out my credentials into a credential file, but store them using aws-keychain.  Would it be possible to have an option to not have `profile` mandatory?

**Comment by josegonzalez** [comment]
Yep it would. Do you mind whipping up a pull request for this? I'm not familiar with the aws-keychain.

**Comment by mbrannigan** [comment]
Wouldn't mind at all.  Thanks!

**Comment by woqer** [comment]
This would be good. I was annoyed that I have to put my credentials on a profile file, since I am already using `awsam` to manage different AWS accounts through ENV variables

**Comment by josegonzalez** [comment]
@woqer pull requests welcome.

**Comment by woqer** [comment]
Not sure about `aws-keychain`, but this PR works for `awsam` and any other way in which boto takes credentials https://github.com/garnaat/kappa/pull/104 @josegonzalez

**Comment by leoamigood** [comment]
do you think we could also use environment vars like (AWS_SECRET_ACCESS_KEY and AWS_SECRET_KEY_ID) instead of profile if available?

**Comment by wvidana** [comment]
Yes, that should work on the new release that got out recently
